charts/zabbix/templates/serviceaccount-ha-helper.yaml

```diff
@@ -5,11 +5,7 @@ kind: ServiceAccount
 metadata:
   name: {{ template "zabbix.fullname" . }}-ha-helper
   labels:
-    app: {{ template "zabbix.fullname" . }}-ha-helper
-    app.kubernetes.io/name: zabbix-server
-    helm.sh/chart: {{ include "zabbix.chart" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-server
-    app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-server
+    {{- include "zabbix.labels" . | nindent 4 }}
```

Comment:
Hi @fibbs!

Is only  this labbels sufficient to reference by zabbix-server and HA files?

Reply:
I think so, as for `ServiceAccounts` there is no label-based matching whatsoever, they are being used by name. That's why I don't have a `app.kubernetes.io/component` label added to serviceaccounts.
Or did I get your question wrong?

Reply:
You understood correctly. Thank you for explaining this.